### ISSUE TITLE
Lead Engineer Engineering Velocity (Remote)

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#7180]
Lead Engineer, Engineering Velocity (Remote)<br>
https://ift.tt/2vuXs61<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0001/8284/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> Remote<br>
<strong>URL:</strong> <a href="https://boards.greenhouse.io/invision/jobs/1101794#.WtoMVNPwbq0">https://boards.greenhouse.io/invision/jobs/1101794#.WtoMVNPwbq0</a></p>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">InVision is the Digital Product Design platform used to make the world&rsquo;s best customer experiences. We provide design tools and educational resources for teams to navigate every stage of the product design process, from ideation to development.</span></div>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Today, more than 3.5 million people use InVision to create a repeatable and streamlined design workflow; rapidly design and prototype products before writing code, and collaborate across their entire organization. That includes more than 80 percent of the Fortune 100, and organizations like Airbnb, Amazon, HBO, Netflix, Slack, Starbucks and Uber, who are now able to design better products, faster.</span></div>
<div>&nbsp;</div>
<div>
<span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">InVision is a fully distributed company with employees in 25 countries around the world. The company has raised more than $235 million in funding from leading investors including Accel, ICONIQ, FirstMark, Tiger Global, Battery Ventures, and Spark Capital. Visit us at</span><span class="attrlink url author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z"><a class="attrlink" href="https://www.invisionapp.com/">&nbsp;InVisionApp.com</a></span><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">&nbsp;and</span><span class="attrlink url author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z"><a class="attrlink" href="https://www.invisionapp.com/blog">InVisionApp.com/blog</a></span><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">.</span>
</div>
<div>&nbsp;</div>
<div>
<span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">The Engineering Velocity team at InVision is on a mission to remove all friction from the development process. We do so by automating all the things from service creation through build, test, deploy, and&nbsp;</span><span class="thread-441057776674352902989358 author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">monitoring.</span><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">&nbsp;Our goal is to make it so fast and easy to do things the Right Way</span>&nbsp;<span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z h-lparen">(</span><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">&trade;) so that there is virtually no barrier to taking ideas all the way out to production. It&rsquo;s like perpetual motion for software development, except that it&rsquo;s not hypothetical.</span>
</div>
<div>&nbsp;</div>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">You&rsquo;ll love this job because you like to delight the people around you by building tools that make their lives easier. You get frustrated with manual processes; why not spend your day making sure they are eliminated one by one? You also love doing this in a supportive environment with engineers who really like working together to achieve amazing things.</span></div>
<div>&nbsp;</div>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Want to know what kind of an impact you can have in this role?</span></div>
<div>&nbsp;</div>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Let&rsquo;s say you&rsquo;re a developer and you want to deploy the latest version of your code across multiple environments. Let&rsquo;s say these services, written in Go, are all running in AWS on a Kubernetes cluster and that you have lots of separate production environments that all need your latest code. Wouldn&rsquo;t it be cool if you could jump into Slack and say:</span></div>
<div>&nbsp;</div>
<div><span class="ace-all-bold-hthree"><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z"><strong>deploy my-service to production</strong></span></span></div>
<div>&nbsp;</div>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">And it just works. In parallel. Super fast, reliable, and with no hand-holding.</span></div>
<div>&nbsp;</div>
<div><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Does that sound like fun? Well, sorry. We already did that. But guess what, we have tons more innovative ideas like that one and we know you&rsquo;d bring a lot of your ideas to the table as well.</span></div>
<div>&nbsp;</div>
<div>
<span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">We know you care about removing friction everywhere you can and building really&nbsp;</span><span class="thread-584570341435490788638420 author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">easy-to-use t</span><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">ools that are loved by developers. If this sounds like you, please send us a cover letter and a r&eacute;sum&eacute; telling us about all the cool things you&rsquo;ve done and why this job description sounds like a great fit for you.</span>
</div>
<div>&nbsp;</div>
<div><span class="ace-all-bold-hthree"><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z"><strong>Responsibilities:</strong></span></span></div>
<ul>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Provide services that are used by all InVision's products and teams</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Continuously deliver solutions via Go, Kafka, MongoDB, Docker, Kubernetes, and more</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Operate your services in production</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Ensure your software is well-designed, well-tested, and scales to millions of users</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Ensure you and your teammates are following good development practices and getting better every day</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Participate in Code Reviews and provide feedback to your fellow engineers</span></li>
</ul>
<div>&nbsp;</div>
<div><span class="ace-all-bold-hthree"><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z"><strong>Requirements:</strong></span></span></div>
<ul>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Experience with operating containers in production, familiarity with container orchestration.</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Demonstrates a high degree of personal accountability and is a team multiplier</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Loves to code, to write tests, and to problem solve and debug</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Learns continuously</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Creates a positive environment for the people around you</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Designs performant systems for high throughput workloads</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Delivers elegant APIs that are easy to use</span></li>
</ul>
<div>&nbsp;</div>
<div><span class="ace-all-bold-hthree"><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z"><strong>Nice to haves:</strong></span></span></div>
<ul>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Experience with Kubernetes in production.</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Experience building internal tools.</span></li>
<li><span class=" author-d-4z65zz66zl57z75zyiz66zfr2fz87zwz89znujo5g9z84z06z70zwmz69zz78zvez83zlz122zz86zuz82z0z78zz76zz78zlr44z90z">Experience setting up / managing a CI/CD solution.</span></li>
</ul>
<p><strong>To apply:</strong> <a href="https://boards.greenhouse.io/invision/jobs/1101794#.WtoMVNPwbq0">https://boards.greenhouse.io/invision/jobs/1101794#.WtoMVNPwbq0</a></p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
April 20, 2018 at 11:15AM
